packages/docs-page/style.module.css

```
@@ -104,7 +104,8 @@ which is rendered into docs-sidenav */
   padding: 0 1rem;
 
   @media (--desktop-viewports) {
-    flex: 1;
+    flex-grow: 1;
+    flex-shrink: 1;
```

I'm honestly having trouble understanding all the nuance both of flex-basis and how it has changed over the years (https://developer.mozilla.org/en-US/docs/Web/CSS/flex-basis#values). But taking a more pragmatic approach, the CSS below seems to work both in FireFox 78 and in newer browsers:

```suggestion
    flex: 1 1 auto;
```

As an alternate solution, this also seems to work:

```suggestion
    flex: 1 1;
    min-width: 0;
```